lib/core/src/lib/card-view/components/card-view-keyvaluepairsitem/card-view-keyvaluepairsitem.component.ts

```diff
@@ -22,10 +22,11 @@ import { MatTableDataSource } from '@angular/material/table';
 import { BaseCardView } from '../base-card-view';
 
 @Component({
-    selector: 'adf-card-view-keyvaluepairsitem',
+    selector: 'adf-card-view-key-value-pairs-item',
```

Comment:
should we also rename files names as they have the same typo?

Reply:
the component is called textitem, so the style resembles the naming, changing the name might cause extra troubles.
cc: @DenysVuika